src/Modal.js

```diff
@@ -96,6 +96,15 @@ const propTypes = {
    */
   restoreFocus: PropTypes.bool,
 
+  /**
+   * Options passed to focus function when `restoreFocus` is set to `true`
+   *
+   * @link  https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus#Parameters
+   */
+  restoreFocusOptions: PropTypes.shape({
```

Comment:
does TS not bundle a definition for this?

Reply:
It does provide a `FocusOptions` interface. Checkout the updated [`types/components/Modal.d.ts`](https://github.com/react-bootstrap/react-bootstrap/pull/3563/files#diff-1b665d8d1834a0d7a873f2b96d9080db) file.